[2024-05-29T19:45:58.739+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: 1_dag.is_api_available manual__2024-05-29T19:45:36.948855+00:00 [queued]>
[2024-05-29T19:45:58.786+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: 1_dag.is_api_available manual__2024-05-29T19:45:36.948855+00:00 [queued]>
[2024-05-29T19:45:58.794+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-05-29T19:45:58.804+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-05-29T19:45:58.805+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-05-29T19:45:58.908+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2024-05-29 19:45:36.948855+00:00
[2024-05-29T19:45:58.944+0000] {standard_task_runner.py:55} INFO - Started process 6143 to run task
[2024-05-29T19:45:58.963+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', '1_dag', 'is_api_available', 'manual__2024-05-29T19:45:36.948855+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/1_dag.py', '--cfg-path', '/tmp/tmp3lhcrmiw']
[2024-05-29T19:45:59.349+0000] {standard_task_runner.py:83} INFO - Job 29: Subtask is_api_available
[2024-05-29T19:46:00.008+0000] {task_command.py:376} INFO - Running <TaskInstance: 1_dag.is_api_available manual__2024-05-29T19:45:36.948855+00:00 [running]> on host 4b56679d13b8
[2024-05-29T19:46:00.516+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=1_dag
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2024-05-29T19:45:36.948855+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-05-29T19:45:36.948855+00:00
[2024-05-29T19:46:00.521+0000] {http.py:120} INFO - Poking: api/
[2024-05-29T19:46:00.588+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2024-05-29T19:46:00.611+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2024-05-29T19:46:01.042+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2024-05-29T19:46:01.093+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=1_dag, task_id=is_api_available, execution_date=20240529T194536, start_date=20240529T194558, end_date=20240529T194601
[2024-05-29T19:46:01.177+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-05-29T19:46:01.702+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
